pkg/integrations/v2/metricsutils/versionshim.go

```diff
@@ -18,49 +18,49 @@ import (
 // CreateShim creates a shim between the v1.Config and v2.Config. The resulting
```

Comment:
I'd like to avoid exposing the things here; see my tips in the test file for how we can do that.

Reply:
I change the struct from being exposed but I still need to expose the fields for marshaling, seem reasonable?

Reply:
Hmm, I'm not sure I understand why it's needed. Weren't they being marshaled before to /-/config without the fields being exported?

Reply:
Good call here, found the underlying issue with a bug in my code where the v1config was not nil so it wasn't exported correctly.